packages/commercetools/api-client/src/helpers/customer/index.ts

```diff
@@ -0,0 +1,11 @@
+export const changeCustomerEmail = (email: string) => ({
```

Comment:
for what we need those? Is this a convention in the api client? I don't think these functions bring any value by being externalized, they're one-liners

Reply:
@filrak this is an internal helper, it's not exposed to the external world

Reply:
@filrak they are action creator actually here is the other example: https://github.com/DivanteLtd/vue-storefront/blob/next/packages/commercetools/api-client/src/helpers/cart/actions.ts, @filipsobol please ad *Action ad the end.

Reply:
i know its a private api but I'm askig about the reason behind externalziing a one line fn into a separate file?

Reply:
IMO in this case it only overcomplicates things.